routes/api-routes.js

```diff
@@ -58,6 +58,34 @@ module.exports = function (app) {
     }
   });
 
+  // Route for getting some data about our user to be used client side
+  app.get("/api/user_data/:id", (req, res) => {
+    if (!req.user) {
+      // The user is not logged in, send back an empty object
+      res.json({});
+    } else {
+      db.User.findOne({
+        where: {
+          id: req.params.id
+        }
+      })
+        .then(data => {
+          res.json({
+            // adding user data from DB structure
+            id: data.id,
+            email: data.email,
+            name: data.name,
+            soberSince: data.soberSince,
+            stars: data.stars,
+            nextBadge: data.nextBadge
+          });
+        })
+        .catch(err => {
+          console.log(err);
+        });
+    }
+  });
+
```

Comment:
Hey Molly!
Everything looks good, but I have a question about this highlighted section.
I'm not sure if you can see it on your end, but on my end it looks like you are adding line 61-88 and there is very similar code above in lines 41-59. Are you seeing this duplication on your side too? And if so is 61-88 something you've added? Does it replace 41-59 and can you delete 41-59?

Reply:
Hi Keith!

The code I added was to get one user by ID. I wasn't sure about the one above it (41-59) - was that to get data for multiple users? Maybe I'm misunderstanding, and the one above gets just the one user also. But I thought that to get the one user, we need to pass in a parameter.